os/include/tinyara/pm/pm.h

```diff
@@ -199,6 +199,18 @@ enum pm_state_e {
 	PM_COUNT,
 };
 
+/* This enumeration provides all power management related wakeup source code. */
+
+typedef enum {
+	PM_WAKEUP_BLE,           /* BLE Interrupts */
+	PM_WAKEUP_WIFI,          /* Wifi Interrupts */
+	PM_WAKEUP_LOG_UART,      /* LOG UART Interrupts */
+	PM_WAKEUP_UART,          /* UART Interrupts */
+	PM_WAKEUP_GPIO,          /* Audio Interrupts */
```

Comment:
Not only audio use GPIO but also other module will use GPIO. Hence GPIO Interrupts is proper.